modin/config/envvars.py

```diff
@@ -109,14 +109,12 @@ def _get_default(cls):
                 )
             return "Dask"
         try:
-            from omniscidbe import PyDbEngine  # noqa
-        except ModuleNotFoundError:
-            try:
-                from dbe import PyDbEngine  # noqa
-            except ImportError:
-                pass
-            else:
-                return "Native"
+            from modin.experimental.core.execution.native.implementations.omnisci_on_native.utils import (
+                DBEngine,
+            )
+            _ = DBEngine.get_instance()
```

Comment:
I don't understand why did you convert this to singleton.
By default, modules and classes are already singletons in Python (you can technically overcome this by doing some dirty hacks, but we aren't doing anything of this).

Reply:
I know, it is just tries to bypass the issue. Let's continue the discussion above.